### PR TITLE
Update roles for jobs

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Enhanced the ldap sync job template to include tightly scope roles.

#### How do we test this?
Follow the LDAP Group Sync section of `jobs/README.md`

cc: @redhat-cop/day-in-the-life-ops

### CODE REVIEW

File: jobs/README.md [hunk 1]
```diff
@@ -32,36 +32,57 @@ oc process -v=JOB_SERVICE_ACCOUNT=pruner -f <template_file> | oc create -f-
 
 ### LDAP Group Synchronization
 
-The [scheduledjob-ldap-group-sync.json](scheduledjob-ldap-group-sync.json) facilitates routine [LDAP Group Synchronization](https://docs.openshift.com/container-platform/3.4/install_config/syncing_groups_with_ldap.html) synchronize groups defined in an LDAP store with OpenShift's internal group storage facility. 
+The [scheduledjob-ldap-group-sync.json](scheduledjob-ldap-group-sync.json) facilitates routine [LDAP Group Synchronization](https://docs.openshift.com/container-platform/3.4/install_config/syncing_groups_with_ldap.html) synchronize groups defined in an LDAP store with OpenShift's internal group storage facility.
```
Comment (sabre1041) [inline jobs/README.md]: Can the latest docs be referenced?

File: jobs/README.md [hunk 2]
```diff
@@ -32,36 +32,57 @@ oc process -v=JOB_SERVICE_ACCOUNT=pruner -f <template_file> | oc create -f-
...
 
-Prior to instantiating the template, the following must be completed within a project:
-
-1. Create a new service account
+This template makes several assumptions about your LDAP architecture and intentions of your group sync process, and is meant to showcase a common use case seen in the field. In this case, we use a top level group to designate all users and groups that will have access to OpenShift. We then create child groups to designate users who should have certain capabilities in OpenShift. A sample tree structure might look like:
```
Comment (sabre1041) [inline jobs/README.md]: Maybe a callout that customizations will most likely be required by the end user

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Looks good. Added a few minor items

**Comment by sabre1041** [comment]
@etsauer I validated the functionality. But as discussed offline, we probably need to call out the fact that this example _must_ be created in a project called `cluster-ops` as the ClusterRoleBinding has this value hardcoded. 

OpenShift 3.7 is targeted to have the functionality to have the current namespace parameter present ([Trello](https://trello.com/c/zcRQFri0/916-(2)-%24%7Bnamespace%7D-template-parameter-%5Btemplates%5D))

**Comment by etsauer** [comment]
@sabre1041 done.

**Review by sabre1041 — APPROVED** [review]

**Pull request merged**